onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -260,9 +260,16 @@ void RegisterConvActivationFusionRules(SelectorActionRegistry& registry) {
   const auto name = "ConvAct";
   auto action = std::make_unique<actions::FuseConvActivationAction>();
 #if !defined(ORT_MINIMAL_BUILD)
+  SelectorActionRegistry::OpVersionsMap op_versions_map;
```

Comment:
Does FuseConvActivationAction need to be updated to return kMSInternalNHWCDomain if the the target node is in that domain?

Reply:
The implementation of the class needs to be modified if that is what you are asking; currently it only work for Conv op in ONNX domain (target node op FusedConv in MsDomain). FuseConvActivationAction needs to supply the target domain. It does not come from anywhere else. The object itself doesn't need to be updated. However when we create this object we create it for specific domain otherwise no why to make it work different domains. Conv op node from kOnnxDomain will be translated to FusedConv in kMsDomain as you already know. NhwcConv op from kMsInternalNhwcDomain goes to FusedNhwcConv in kMsInternalNhwcDomain.